muikku-core-plugins/src/main/java/fi/muikku/plugins/workspace/WorkspaceMaterialsBackingBean.java

```diff
@@ -54,7 +59,7 @@
   private SystemSettingsController systemSettingsController;
 
   @RequestAction
-  public String init() {
+  public String init() throws UnsupportedEncodingException {
```

Comment:
This can cause an EJBRemovedException, remove throws and replace with return NavigationRules.INTERNAL_ERROR && log